WordPress/src/main/java/org/wordpress/android/models/Role.java

```diff
@@ -21,21 +27,42 @@
         this.backgroundColor = backgroundColor;
     }
 
-    public static String toString(Context context, Role role) {
+    // Label to be used in UI
+    public static String getLabel(Context context, Role role) {
         return context.getString(role.label);
     }
 
-    // This method will be used to determine the role of the user from network request
-    public static Role fromString(String value) {
+    // Returns the key for that role
+    public static String toKey(Role role) {
+        switch (role) {
+            case SUPER_ADMIN:
+                return KEY_SUPER_ADMIN;
+            case ADMIN:
+                return KEY_ADMIN;
+            case EDITOR:
+                return KEY_EDITOR;
+            case AUTHOR:
+                return KEY_AUTHOR;
+            case CONTRIBUTOR:
+                return KEY_CONTRIBUTOR;
+            default:
```

Comment:
Aha, I see. Had a look at [#03002cc](https://github.com/wordpress-mobile/WordPress-Android/pull/3916/commits/03002cc4e2dfb94a8d2fcf29d3048a3b70a405da) and here's the thing:
*  The Role enum is defined in that same module and so we can always have perfect knowledge of the instances defined. Technically, the `toKey(Role)` method can never receive an "unknown" case and so the switch can always have coverage for all the cases (and that's why leaving out the `default` can help by making the compiler/linter tools stop if the switch is not complete).
* The only case not covered yet is when the input parameter is `null`. I understand what you mean about trying to avoid check for NPE at the caller but that's Java... You need to cover the `null` case as well and if you want to avoid returning null, the other way around is to throw. Ofc you can return a valid string even when input is null as you have in the new commit but you need to consider whether that would be correct or it would just help hide bugs.